smart-action/src/main/java/org/smartdata/action/HelloAction.java

```diff
@@ -20,12 +20,12 @@
 import org.smartdata.action.annotation.ActionSignature;
```

Comment:
One pull request should only deal with one issue. Please submit two pull requests instead.